packages/celotool/src/cmds/account/verify.ts

```diff
@@ -1,8 +1,11 @@
 import { AccountArgv } from '@celo/celotool/src/cmds/account'
 import { portForwardAnd } from '@celo/celotool/src/lib/port_forward'
-import { CeloContract, newKit } from '@celo/contractkit'
-import { AttestationsWrapper } from '@celo/contractkit/lib/wrappers/Attestations'
-import { ActionableAttestation, decodeAttestationCode } from '@celo/walletkit'
+import { CeloContract, ContractKit, newKit } from '@celo/contractkit'
+import {
+  ActionableAttestation,
+  AttestationsWrapper,
+} from '@celo/contractkit/lib/wrappers/Attestations'
+import { decodeAttestationCode } from '@celo/walletkit'
```

Comment:
Can this be easily moved to contractkit?

Reply:
Moved it to utils!